### ISSUE TITLE
change command

### ISSUE THREAD

**Issue by kotviphp** [respawner/looking-glass#137]
How change command?

defult,  ping count 10 8.8.8.8 source lo0,  these are the wrong commands
there should be a command ping router 24 and then ip address

**Comment by kotviphp** [comment]
![LG](https://user-images.githubusercontent.com/25497579/92918042-e41eb480-f450-11ea-88e7-4524d328e556.JPG)

**Comment by respawner** [comment]
Commands are coded and you cannot changed them without changing the source code.

Please give all the details for this issue because we don't know what kind of router you are talking about and therefore canot help you. Try to keep issues for actual bugs and feature requests as well, if you have any questions please join the Slack channel.

**Comment by kotviphp** [comment]
TiMOS-C-16.0.R10 cpm/hops64 Nokia 7750 SR Copyright (c) 2000-2019 Nokia

**Comment by respawner** [comment]
Could you please provide examples for commands which do not work on your Nokia devices?

**Comment by kotviphp** [comment]
ping router  24 8.8.8.8  
as an example

**Comment by gya235** [comment]
"router 24" is a virtual context (VPN) inside router . By deafult the context is 1 but we need  context 24

**Comment by gya235** [comment]
This is in real envaironment issue:


A:BSR01.SNBAV.ALM# ping router
router              router-instance
A:BSR01.SNBAV.ALM# ping router 24 8.8.8.8
PING 8.8.8.8 56 data bytes
64 bytes from 8.8.8.8: icmp_seq=1 ttl=109 time=75.7ms.
64 bytes from 8.8.8.8: icmp_seq=2 ttl=109 time=75.6ms.
64 bytes from 8.8.8.8: icmp_seq=3 ttl=109 time=75.6ms.
64 bytes from 8.8.8.8: icmp_seq=4 ttl=109 time=75.6ms.
64 bytes from 8.8.8.8: icmp_seq=5 ttl=109 time=75.6ms.

---- 8.8.8.8 PING Statistics ----
5 packets transmitted, 5 packets received, 0.00% packet loss
round-trip min = 75.

**Comment by respawner** [comment]
So `router X` is only to be used when a VRF is needed?

**Comment by gya235** [comment]
Yes. And Internet instance still use another VRF context not global. It`s a common practice. That`s why we should point vrf  instance  for those command (ping, trace, show  bgp and so on)

**Comment by kotviphp** [comment]
will be fixed?
